Makefile

```diff
@@ -20,4 +20,8 @@ lint:
 test:
 	cask exec ert-runner
 
+magit-test: elpa
```

Comment:
Pulled this over from evil-magit, ideally, we'd be able to move this under test: somehow.

Reply:
Seems like there should be a discussion about how to run tests for each package generally.

Reply:
I don't really have any thoughts on this. Open to thoughts from others.

Reply:
I tweaked the test targets a little, magit-test will run magit tests and test: will run the only two tests we have now. Not the most polished but good enough, probably..

When I ran the tests, I did end up getting an error but I get the same error in the original evil-magit repo too.

```
Test evil-magit-section-map-tests condition:
    (ert-test-failed
     ((should
       (eq
	(lookup-key ... ...)
	(nth 2 binding)))
      :form
      (eq magit-diff-visit-worktree-file magit-diff-visit-file-worktree)
      :value nil))
   FAILED  4/5  evil-magit-section-map-tests
   passed  5/5  evil-magit-section-maps-accounted-for

Ran 5 tests, 4 results as expected, 1 unexpected (2019-10-20 16:02:10-0700)

1 unexpected results:
   FAILED  evil-magit-section-map-tests

make: *** [test] Error 1```